google/cloud/security/common/gcp_type/firewall_rule.py

```diff
@@ -326,6 +369,9 @@ def __lt__(self, other):
         """
         return (self.action == other.action and
                 (other.applies_to_all or
+                 self.any_value or
```

Comment:
I recommend keeping the order the same, so you always check self.any_value then other.any_value then (self|other).applies_to_all, then the expanded rules checks.

Reply:
Done